src/components/Form/form.scss

```diff
@@ -0,0 +1,21 @@
+.inner-form-wrapper {
+  display: flex;
+  justify-content: center;
+  &.form-and-result {
+    justify-content: space-between;
+  }
+  form,
+  .results-only {
+    width: 500px;
+  }
+
+  .input-and-results {
+    margin-left: 40px;
+  }
+
+  > div {
+    h1 {
+      margin-top: 30px;
```

Comment:
Do you have thoughts on `px` vs something more responsive, such as `em` or `rem`? I still mostly follow [this pattern](https://bits.theoremone.co/css-pro-tips-responsive-font-sizes-and-when-to-use-which-units/).

Reply:
Either way, we probably want to use the `var(--spacing-*)` variables in this code